tests/test_application.py

```diff
@@ -3,24 +3,29 @@
 import os
 
 from jupyterhub import orm
+from tests.temp_mixin import TempMixin
 from tornado import gen, testing
 
-from remoteappmanager.application_config import ApplicationConfig
 from remoteappmanager.application import Application
 from remoteappmanager.docker.container import Container
 from remoteappmanager.docker.container_manager import ContainerManager
+from tests import utils
 
 
-class TestApplication(testing.AsyncTestCase):
+class TestApplication(TempMixin, testing.AsyncTestCase):
```

Comment:
Instead of mixin, I would prefer adding a method for removing the temporary directory and do `addCleanUp` in the setup, unless `TempMixin` is used often.